vendor/plugins/edu.harvard.dce.paella.timedCommentsPlugin/timedCommentsHeatmap.js

```diff
@@ -16,7 +16,7 @@ paella.addPlugin(function () {
       this.ifModifiedSinceClientOffset = null;
       this.isEnabled = false;
     }
-    
+
     getAlignment() {
```

Comment:
JSDoc? But this is not part of your PR

Reply:
That would be a whole other can of worms to start adding JSDocs here. These plugins confirm with Paella 6 that doesn't have JDocs. The Paella 7 will require all these plugins to be revisited anyway.

Reply:
Okay